### ISSUE TITLE
Screen Reader Issue With Changing a Post's Date

### ISSUE THREAD

**Issue by Access4All96** [WordPress/gutenberg#7991]
I use a desktop running Mozilla Firefox. I also use a screen reader called NVDA. I noticed if I want to change a post's date using Gutenberg, I am unable to do this with my screen reader. 
The version of Gutenberg I am using is version 3.2.0. 
**To Reproduce**
1. Create a new post. 
2. I go to the button titled "Open Publish Panel."
3. I go to the item that shows me the current date and time. I press enter to expand this part, which shows me controls to change the date and time of my post. 
4. I am given this calendar view, but there is no way for me to change the date. NVDA treats this part as a list, but when I press enter to focus on this "list," I can not use the arrow keys to change any values. 

**Expected behavior**
As soon as I use my arrow keys to navigate the calendar view, it should read out what day I am highlighting.

**Desktop**
 - OS: Windows 10
 - Browser: Mozilla Firefox
 - Version 61.0.1

**Comment by designsimply** [comment]
Thank you for reporting this! There is already a bug report about date picker accessibility which addresses this issue at #1311. I adding a screenshot here (including showing some of the rendered HTML) as a reference and closing this issue as a duplicate of #1311.

<img src="https://user-images.githubusercontent.com/1119271/42833726-38acfb12-89b2-11e8-8aa5-09896a139ed9.png" width="270" alt="screen shot 2018-07-17 at tue jul 17 11 01 52 am">
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=13999&action=edit running WordPress 4.9.7 and Gutenberg 3.2.0 using Firefox 61.0.1 on macOS 10.13.5.</sup>
